features/edition-images.feature

```diff
@@ -52,3 +49,19 @@ Feature: Images tab on edit edition
     And I click the "Save and continue" button on the preview page
     And I update the image details and save
     Then the publication "Standard Beard Lengths" should have 1 image attachment
+
+  Scenario: Small image uploaded
+    And I have the "Preview images update" permission
+    And I start drafting a new publication "Standard Beard Lengths"
+    When I am on the edit page for publication "Standard Beard Lengths"
+    And I navigate to the images tab
+    And I upload a 64x96 image
+    Then I should get the error message "Images image data file must be 960px wide and 640px tall, but is 64px wide and 96px tall"
```

Comment:
This is not the prettiest of error messages – but you've already said you're going to follow up with some more changes to improve error messaging 👍🏻